app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    any_issues = False
```

Comment:
There is no need for this 'any_issues' variable. Remove it and simply `return f"Friends can go to {cafe.name}"` at the end.